src/main/java/MainClass.java

```diff
@@ -1,9 +1,13 @@
+import gui.RootPane;
+
 public class MainClass {
 
     /**
      *  @param args - the main arguments.
      */
     public static void main(String[] args) {
+        RootPane root = new RootPane();
+        root.launch(RootPane.class, args);
```

Comment:
Is this really the only way to launch the rootpane? If so, could you place some comments explaining why since it's not the most trivial thing